core/Config.php

```diff
@@ -426,47 +412,24 @@ protected function encodeValues($values)
     public function &__get($name)
     {
         if (!$this->initialized) {
-            $this->init();
+            $this->reload(array($this->pathGlobal, $this->pathCommon), $this->pathLocal);
 
             // must be called here, not in init(), since setTestEnvironment() calls init(). (this avoids
             // infinite recursion)
-            Piwik::postTestEvent('Config.createConfigSingleton',
-                array($this, &$this->configCache, &$this->configLocal));
+            // TODO: Removed last argument, must check it's ok
```

Comment:
TODO was to check if pro plugin tests pass, they do. Just FYI for why I removed the // TODO